README.md

```diff
@@ -83,3 +84,13 @@ class Command(BaseCommand):
                         old_cache.delete(actual_key)
 
 ```
+
+# Scheduler related usage
+You can use some extra functionality provided by this plugin, if secure cache settings are not added in the `default` settings, please make sure that you specify your secure cache settings name by setting `DJANGO_REDIS_SECURE_CACHE_NAME` settings. To enable these feature, this simply by adding `@secure_redis.rq_job` decorator on top of your task. By doing this you can get use of:
+
+1. `delay` method, so you can use it like `my_job.delay()`. This method is same as `django_rq.job.delay`
```

Comment:
1. A `delay` method, which can be used when calling the task method (ex: `my_task.delay()`). This method has the same functionality as `django_rq.job.delay`